### ISSUE TITLE
feature request: When copying an accessibility ID that contains a newline, please use "\n" instead of the actual newline character. This will make it easier to paste the ID directly into code. 

### ISSUE THREAD

**Issue by miwtoo** [appium/appium-inspector#1440]
### Current Behavior

```
Setting
Tab 3 of 3
```

### Suggested Solution

```
Setting\nTab 3 of 3
```

### Additional Information
When copying an accessibility ID that contains a newline, please use "\n" instead of the actual newline character. This will make it easier to paste the ID directly into code. 

<img width="516" alt="image" src="https://github.com/appium/appium-inspector/assets/12011082/c858de31-dd90-4898-ad9c-376b8eb1382a">

accessibility id : 
<img width="1212" alt="image" src="https://github.com/appium/appium-inspector/assets/12011082/038ae6d6-7e58-4160-8a58-3e368db72d1e">

**Comment by miwtoo** [comment]
@eglitise 
```XML
<?xml version='1.0' encoding='UTF-8' standalone='yes' ?>
<hierarchy index="0" class="hierarchy" rotation="0" width="1008" height="2077">
  <android.widget.FrameLayout index="0" package="com.example.automation_appium" class="android.widget.FrameLayout" text="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2244]" displayed="true">
    <android.widget.LinearLayout index="0" package="com.example.automation_appium" class="android.widget.LinearLayout" text="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2190]" displayed="true">
      <android.widget.FrameLayout index="0" package="com.example.automation_appium" class="android.widget.FrameLayout" text="" resource-id="android:id/content" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2190]" displayed="true">
        <android.widget.FrameLayout index="0" package="com.example.automation_appium" class="android.widget.FrameLayout" text="" checkable="false" checked="false" clickable="false" enabled="true" focusable="true" focused="true" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2190]" displayed="true">
          <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2190]" displayed="true">
            <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2190]" displayed="true">
              <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2190]" displayed="true">
                <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2190]" displayed="true">
                  <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2042]" displayed="true">
                    <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2042]" displayed="true">
                      <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,2042]" displayed="true">
                        <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,0][1008,1860]" displayed="true">
                          <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" content-desc="Which brand of water&#10;would you like to buy?" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[227,113][781,266]" displayed="true" />
                          <android.widget.EditText index="1" package="com.example.automation_appium" class="android.widget.EditText" text="" resource-id="" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,294][1008,802]" displayed="true" hint="Search by water name...&#10;Top Waters :">
                            <android.widget.HorizontalScrollView index="0" package="com.example.automation_appium" class="android.widget.HorizontalScrollView" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="true" selected="false" bounds="[67,563][941,671]" displayed="true">
                              <android.widget.Button index="0" package="com.example.automation_appium" class="android.widget.Button" text="" content-desc="Nestlé Pure Life" resource-id="" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[89,563][426,671]" displayed="true" />
                              <android.widget.Button index="1" package="com.example.automation_appium" class="android.widget.Button" text="" content-desc="Aquafina" resource-id="" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="true" bounds="[449,563][736,671]" displayed="true" />
                              <android.widget.Button index="2" package="com.example.automation_appium" class="android.widget.Button" text="" content-desc="Evian" resource-id="" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[758,563][941,671]" displayed="true" />
                            </android.widget.HorizontalScrollView>
                          </android.widget.EditText>
                          <android.view.View index="2" package="com.example.automation_appium" class="android.view.View" text="" content-desc="Thai Water" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[45,802][229,854]" displayed="true" />
                          <android.widget.HorizontalScrollView index="3" package="com.example.automation_appium" class="android.widget.HorizontalScrollView" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="true" selected="false" bounds="[11,865][997,1320]" displayed="true">
                            <android.widget.ImageView index="0" package="com.example.automation_appium" class="android.widget.ImageView" text="" content-desc="Crystal&#10;฿5.33" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[11,865][522,1320]" displayed="true" />
                            <android.widget.ImageView index="1" package="com.example.automation_appium" class="android.widget.ImageView" text="" content-desc="Singha&#10;฿10.00" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[522,865][997,1320]" displayed="true" />
                          </android.widget.HorizontalScrollView>
                          <android.view.View index="4" package="com.example.automation_appium" class="android.view.View" text="" content-desc="Others Water" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[45,1331][271,1383]" displayed="true" />
                          <android.widget.HorizontalScrollView index="5" package="com.example.automation_appium" class="android.widget.HorizontalScrollView" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="true" selected="false" bounds="[11,1394][997,1849]" displayed="true">
                            <android.widget.ImageView index="0" package="com.example.automation_appium" class="android.widget.ImageView" text="" content-desc="Aquafina&#10;฿14.00" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[11,1394][522,1849]" displayed="true" />
                            <android.widget.ImageView index="1" package="com.example.automation_appium" class="android.widget.ImageView" text="" content-desc="Acqua Panna&#10;฿79.00" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[522,1394][997,1849]" displayed="true" />
                          </android.widget.HorizontalScrollView>
                        </android.view.View>
                      </android.view.View>
                    </android.view.View>
                  </android.view.View>
                  <android.view.View index="1" package="com.example.automation_appium" class="android.view.View" text="" resource-id="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,2042][1008,2190]" displayed="true">
                    <android.view.View index="0" package="com.example.automation_appium" class="android.view.View" text="" content-desc="Dashboard&#10;Tab 1 of 3" resource-id="" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="true" bounds="[0,2042][336,2190]" displayed="true" />
                    <android.view.View index="1" package="com.example.automation_appium" class="android.view.View" text="" content-desc="Profile&#10;Tab 2 of 3" resource-id="" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[336,2042][672,2190]" displayed="true" />
                    <android.view.View index="2" package="com.example.automation_appium" class="android.view.View" text="" content-desc="Setting&#10;Tab 3 of 3" resource-id="" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[672,2042][1008,2190]" displayed="true" />
                  </android.view.View>
                </android.view.View>
              </android.view.View>
            </android.view.View>
          </android.view.View>
        </android.widget.FrameLayout>
      </android.widget.FrameLayout>
    </android.widget.LinearLayout>
    <android.view.View index="2" package="com.example.automation_appium" class="android.view.View" text="" resource-id="android:id/navigationBarBackground" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" long-clickable="false" password="false" scrollable="false" selected="false" bounds="[0,2190][1008,2244]" displayed="true" />
  </android.widget.FrameLayout>
</hierarchy>
```

**Comment by eglitise** [comment]
Thanks, this confirms that there is an actual newline character in the attribute value. It should be possible to add handling for this.

**Comment by eglitise** [comment]
This fix is included in version 2024.6.1. Thanks again!
